crates/common/fmt/src/dynamic.rs

```diff
@@ -132,18 +132,6 @@ pub fn format_tokens_raw(tokens: &[DynSolValue]) -> impl Iterator<Item = String>
     tokens.iter().map(format_token_raw)
 }
 
-/// Prints slice of tokens using [`format_tokens`] or [`format_tokens_raw`] depending on `json`
```

Comment:
can we keep the helper fn to print tokens and avoid duplicated code above?

Reply:
Done, had to move the helper into the call location due to a circular dependency in `fmt` / `common`